### DISCUSSION TITLE
Map tile sprites do not draw / wrong order

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13729] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The map tile logic seems to be following 1,2,3 patterns until it doesnt around apothic plateau

I noted each tile with a number 
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/2057b3a5-9e46-4515-bbed-f95b0f019136)

Very zoomed out map shows the tiles in order and then it breaks in plateau. Tile 9 doesnt even exist

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/d48aa820-a3ea-4e13-bf18-e31319fca0a3)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/416e68b6-a8f7-431f-80a6-697066692d72)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/57cd5679-2ede-4514-8b90-38eee5d7fe87)

### Reproduction steps

Use this mod (fix the zoom level if needed)

[Mapgenspritetest.zip](https://github.com/FakeFishGames/Barotrauma/files/14888305/Mapgenspritetest.zip)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
Increased width="12000" (from 8000)

It seems that the vanilla map just isnt big enough to display all tiles and cuts one at the plateau, but the display order still seems broken around the middle

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/decf56e3-5745-4dfb-8d5f-f7844068d002)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/0a0f144f-2f5f-4f4a-897a-aa7eff3b21c0)

**Comment by Regalis11** [comment]
This does not actually seem like a bug to me. The tiles are not supposed to be draw in the same order they're in the atlas: different tiles are intended for different biomes. The tiles 1-3 for example are for cold caverns, and seem to correctly repeat until it switches to 4-6 in the plateau.
